pages/common/emacs.md

```diff
@@ -1,24 +1,33 @@
 # emacs
 
 > The extensible, customizable, self-documenting, real-time display editor.
+> See also `emacsclient`.
 > More information: <https://www.gnu.org/software/emacs>.
 
-- Start in console mode (without X window):
+- Start Emacs and open a file:
 
-`emacs -nw`
+`emacs {{path/to/file}}`
 
-- Open a file:
+- Open a file at a specified line number:
 
-`emacs {{path/to/file}}`
+`emacs +{{line_number}} {{path/to/file}}`
 
-- Save a file:
+- Start Emacs in console mode (without X window):
```

Comment:
```suggestion
- Start Emacs in console mode (without an X window):
```

Reply:
I like `without X window` however we'll see

Reply:
`X window` isn't really a name, but a noun, so it should be `the X window`/`an X window`.